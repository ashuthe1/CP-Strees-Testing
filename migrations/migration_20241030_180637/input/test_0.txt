39
-5 -9 3 5 -1 -1 8 4 3 4 5 -6 7 7 3 -4 5 -10 -8 8 -9 -6 9 8 -1 0 5 5 0 0 7 3 1 -2 -3 10 7 -6 1 
97
215 258
571 620
3 651
564 641
569 627
97 458
85 354
75 709
178 746
348 649
38 479
277 440
232 526
302 736
110 387
325 498
160 516
164 258
162 677
660 694
10 465
338 421
235 266
72 495
62 443
353 720
380 413
501 516
125 329
37 106
406 715
302 323
437 466
580 598
234 493
350 698
375 687
338 481
44 705
358 553
492 638
142 710
91 431
87 658
566 759
65 565
64 366
500 759
431 703
156 190
15 539
73 261
283 317
360 742
79 514
716 717
97 647
60 730
169 479
19 146
82 264
327 455
434 448
47 370
108 736
526 750
470 646
182 231
64 624
541 574
349 577
382 673
87 331
255 732
29 750
145 292
51 600
370 491
125 409
80 740
357 516
253 701
23 46
434 542
66 86
335 662
414 716
427 501
250 266
166 627
91 644
14 330
40 463
383 459
214 320
410 707
202 570