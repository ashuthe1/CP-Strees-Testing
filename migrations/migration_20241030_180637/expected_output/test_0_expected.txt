1373
778
12701
1567
1075
8507
7470
12234
9519
3510
10050
2773
4461
5962
6952
1950
6472
2876
8927
556
10586
1231
928
9230
9323
4675
356
15
5614
1889
3927
552
171
317
4186
4392
3789
1861
12963
1613
1757
10133
8431
11090
3339
9829
8240
3632
3320
1002
10685
5859
762
4941
8995
7
10502
12985
6282
3722
5641
1922
59
8705
11551
3422
2071
1509
10888
349
1957
3514
6931
6874
13927
4257
10683
1188
6867
12445
1453
6504
552
368
657
4175
3886
443
391
7729
10670
8698
9829
1012
2836
3753
5676
